addons/web/static/src/core/position_hook.js

```diff
@@ -15,6 +15,7 @@ import { localization } from "@web/core/l10n/localization";
  *  margin?: number;
  *  position?: Direction | Position;
  *  onPositioned?: PositionEventHandler;
+ *  customApplyPosition: (popper: HTMLElement, position: object) => void;
```

Comment:
Do not add another option ;-)
Please use `onPositioned`

Reply:
Okay, I'll try that.